xcm/pallet-xcm/src/lib.rs

```diff
@@ -244,6 +244,12 @@ pub mod pallet {
 		/// The maximum number of local XCM locks that a single account may have.
 		type MaxLockers: Get<u32>;
 
+		/// The maximum number of consumers a single remote lock may have.
+		type MaxRemoteLockConsumers: Get<u32>;
```

Comment:
What does a "lock consumer" mean in this context?

Reply:
its more RemoteLock's Consumers 
there is also a notion of Lock in the pallet, which represent a local lock, which has associated remote locks

Reply:
My question is more geared towards the "consumers" part and not the "lock" part. It was more easily understood when it was named "lock users" but "consumers" is weird -- did you mean "lock provider" instead?

Reply:
I see what you mean, the intention was to use same terms for similar purposes, similar to reference count for account consumers in system pallet.
But users sounds good to me as well.

Reply:
renamed consumers to users

Reply:
@KiChjang can you have a look again please?